pkg/vif/router.go

```diff
@@ -23,12 +22,26 @@ type Router struct {
 	staticOverrides []*routing.Route
 	// The subnets that are currently being routed
 	routedSubnets []*net.IPNet
+	// The subnets that are allowed to be routed even in the presence of conflicting routes
+	whitelistedSubnets []*net.IPNet
```

Comment:
Maybe I would change it for something like allowList instead, especially since we push the project to CNCF. :P

Reply:
@knlambert why would we change that? "whitelisted" is much more concise. It doesn't mean just "allowed", it means that you *override* the fact that it isn't.

Reply:
I'm not sure what the CNCF bit has to do with this? I think this is a whitelist

Reply:
@josecv I agree, it's a whitelist, but nowadays, some people think that by changing the vocabulary, we'll address systemic racism:
https://www.cncf.io/announcements/2021/10/13/inclusive-naming-initiative-announces-new-community-resources-for-a-more-inclusive-future/

Reply:
ah